src/loki/isochrone_action.cc

```diff
@@ -55,11 +65,15 @@ void loki_worker_t::init_isochrones(Api& request) {
     throw valhalla_exception_t{152, std::to_string(max_contours)};
   }
 
-  // validate the contour time by checking the last one
-  const auto contour = options.contours().rbegin();
-  if (contour->time() > max_time) {
+  // check the contour metrics
+  float max_time_c, max_dist_c;
+  std::tie(max_time_c, max_dist_c) = get_max_metrics(options.contours());
```

Comment:
previously this depended on contours being sorted in ascending order. pulling out the max of 2 metrics from a flat "list" we can't do that anymore and might as well allow unsorted contours in the request params..